src/libcommon/log/sentry/handler.cpp

```diff
@@ -265,22 +265,30 @@ void SentryHandler::init(KDC::AppType appType, int breadCrumbsSize) {
         sentry_options_set_environment(options, environment.c_str());
     }
 
+#ifdef NDEBUG
+    sentry_options_set_traces_sample_rate(options, 0.1); // 10% of traces
+#else
+    sentry_options_set_traces_sample_rate(options, 1);
```

Comment:
Is it going to be useful? I mean, is it currently possible to use Sentry in DEBUG mode?